pkg/reconciler/v1alpha1/testing/functional.go

```diff
@@ -112,6 +112,17 @@ func WithRunLatestRollout(s *v1alpha1.Service) {
 	}
 }
 
+// WithRunLatestConfigSpec confgures the Service to use a "runLastest" configuration
+func WithRunLatestConfigSpec(config v1alpha1.ConfigurationSpec) ServiceOption {
+	return func(svc *v1alpha1.Service) {
+		svc.Spec = v1alpha1.ServiceSpec{
+			RunLatest: &v1alpha1.RunLatestType{
+				Configuration: config,
+			},
+		}
+	}
+}
+
 // WithServiceLabel attaches a particular label to the service.
 func WithServiceLabel(key, value string) ServiceOption {
```

Comment:
I think the internal function should probably be extracted as `SetObjectLabels` since, all k8s objects have labels and annotations, and as such it's duplicated code.

Reply:
Do you mean we should rename func `WithServiceLabel` to `SetObjectLabels`?    I don't think I get your comment.

Reply:
I don't think you can do that.
So I meant slightly different thing, but doesn't matter.